cabal-testsuite/PackageTests/JS/JsSources/js-arch.test.hs

```diff
@@ -1,9 +1,10 @@
 import Test.Cabal.Prelude
 
-main = setupAndCabalTest $ do
-    skipUnlessGhcVersion ">= 9.6"
+-- There seems to be no validate workflow for these tests, what are they for?
```

Comment:
Honest question, is this actually being tested anywhere?

Reply:
Due to ` skipUnlessJavaScript`, it can only be tested locally, because our CI doesn't yet build/test on the JS arch, I think. Unless you meant something else than ` skipUnlessJavaScript`?

Reply:
I meant precisely whether this is tested on CI. Or how can I test it locally?

Reply:
I removed the comment. I still don't know how to run this.